### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Meningitix**
German translation

### CODE REVIEW

File: PoGo.NecroBot.CLI/bin/Debug/config/translations/translation.de.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      "Key": "ultraPokeball",
+      "Value": "HyperBall"
+    },
+    {
+      "Key": "masterPokeball",
+      "Value": "MeisterBall"
+    },
+    {
+      "Key": "wrongAuthType",
+      "Value": "unbekannte Authentifizierung in config.json"
```
Comment (VanStorm) [inline PoGo.NecroBot.CLI/bin/Debug/config/translations/translation.de.json]: "unbekannte Authentifizierungsart in der config.json" would be more accurate

File: PoGo.NecroBot.CLI/bin/Debug/config/translations/translation.de.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      "Key": "eventItemRecycled",
+      "Value": "{0}x {1}"
+    },
+    {
+      "Key": "eventPokemonCapture",
+      "Value": "({0}) | ({1}) {2} Lvl: {3} CP: ({4}/{5}) IV: {6}% | Chance: {7}% | {8}m entfernt | mit {9} gefangen ({10} verbleibend). | {11}"
+    },
+    {
+      "Key": "eventNoPokeballs",
+      "Value": "Keine Baelle - Wir haben ein {0} mit {1} CP verpasst."
```
Comment (VanStorm) [inline PoGo.NecroBot.CLI/bin/Debug/config/translations/translation.de.json]: "Bälle" or "Pokébälle" should work fine too since it's utf-8

**Comment by NecronomiconCoding** [comment]
please update the translation according to @VanStorm

File: PoGo.NecroBot.CLI/bin/Debug/config/translations/translation.de.json (hunk 2 above)
Comment (Meningitix) [inline PoGo.NecroBot.CLI/bin/Debug/config/translations/translation.de.json]: Ä macht aber die Konsole nicht ;)

**Pull request closed without merging**